deploy/charts/bitwarden-sdk-server/Chart.yaml

```diff
@@ -21,4 +21,4 @@ version: 0.1.0
 # incremented each time you make changes to the application. Versions are not expected to
 # follow Semantic Versioning. They should reflect the version the application is using.
 # It is recommended to use it with quotes.
-appVersion: "1.16.0"
+appVersion: "v0.3.1"
```

Comment:
appVersion is updated during the release process. It doesn't matter what the version is here. :)

https://github.com/external-secrets/bitwarden-sdk-server/blob/main/.github/workflows/release.yaml#L53

If you check and download the chart from the repository it will be set as such:

```yaml
appVersion: v0.3.1
description: A Helm chart for Kubernetes
name: bitwarden-sdk-server
type: application
version: v0.3.1
```

Why would you like to update it here?

Reply:
I cloned the repo, assuming the Helm chart would work out of the box, but you're right in the end.  :-) 
I just thought maybe someone else might stumble upon this.

I guess it's a close then :-)

Reply:
I mean it's not a problem to set it, I was just wondering what the usecase is. The problem is, that I will keep forgetting to set it on each release. :D That's why automated it. :)

Reply:
Ah you need to run `make helm.test.update`.

Reply:
Sorry for the delay.
Will close it now, thanks for the quick response.